### PR TITLE
fix: blurry dialogs

### PR DESCRIPTION
**Pull request by shyakadavis**
Closes: #53

### CODE REVIEW

**Comment by ablesea** [comment]
Yeah not fixed for me here...

File: src/lib/components/ui/dialog/dialog-content.svelte [hunk 1]
```diff
@@ -21,45 +21,53 @@
...
+		<div class="fixed inset-0 z-50 grid place-items-center">
+			<div
+				bind:this={el}
+				transition:_transition={transitionConfig}
+				use:builderActions={{ builders: [builder] }}
+				{...getAttrs([builder])}
+				{...$$restProps}
+				class={cn(
+					'fixed left-[50%] top-[50%] z-50 grid w-full max-w-lg translate-x-[-50%] translate-y-[-50%] gap-4 border bg-background-100 p-6 shadow-lg sm:rounded-lg md:w-full',
+					className
```
Comment (ablesea) [inline src/lib/components/ui/dialog/dialog-content.svelte:40]: I think the implementation is wrong. I believe using translate was one of the contributing factors.

Comment (shyakadavis) [reply src/lib/components/ui/dialog/dialog-content.svelte:40]: Snap. I have forgotten to remove those class names. Should be fixed now.

Comment (shyakadavis) [reply src/lib/components/ui/dialog/dialog-content.svelte:40]: Noticed the `x` issue. Pushing a fix now.

**Comment by shyakadavis** [comment]
Hey, @ablesea, kindly confirm if it's all good on your end.

**Comment by ieedan** [comment]
Sorry I didn't see this since it was on my work account! It is fixed for me :)

**Pull request merged**